### PR TITLE
[PLAYER-4326] mediaManager.onLoad now triggers play() directly if pla…

### PR DESCRIPTION
**Pull request by zelibobla**
1. mediaManager.onLoad event now respects event.data provided. If .autoplay is true there - player play start directly.
2. Encapsulated cast.receiver.MediaManager methods extending.
3. Fixed some linter errors.

### CODE REVIEW

**Comment by pilievOoyala** [comment]
LGTM

File: lib/src/app/cast.js [hunk 1]
```diff
@@ -18,58 +20,73 @@ import '../styles/player.css';
...
-        logger.info('MediaManager:onPlay', e);
-        Timer.setIdle();
-        mediaManager.origPlay(e);
-    };
+    mediaManager.onPlay = (() => {
+        mediaManager.origOnPlay = mediaManager.onPlay;
+        return (event) => {
+            logger.info('MediaManager:onPlay', event);
+            Timer.setIdle();
+            mediaManager.origOnPlay(event);
```
Comment (marcosmx) [inline lib/src/app/cast.js]: Why this method does not have the same logic to notify the state to the senders?

Comment (zelibobla) [reply lib/src/app/cast.js]: Have the same question. Should be addressed to author of this part:
@aldigun in this commit:
https://github.com/ooyala/chromecast-sample-receiver/commit/4e864e7468d05cbd44907aa85ea6f30171c2f677

**Comment by aldigun** [comment]
For me this [PLAYER-4326] is a sender issue. So I don't understand why we need to have a code change on receiver. Please take a look at my comment on the ticket. We should not merge this without understanding the situation.

**Comment by aldigun** [comment]
I will temporarily close this PR, to be just in case. If this is indeed the solution, we can re-open this.

**Pull request closed without merging**